shared/wallets/send-form/footer/container.js

```diff
@@ -8,6 +8,7 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({})
 
 const mergeProps = (stateProps, dispatchProps) => ({})
 
+// $FlowIssue we're not sending anything
```

Comment:
and here?

Reply:
same, these are all empty which is wrong